### PR TITLE
Asking for confirmation at end of GPSR

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Not tested at all. Modified on GitHub

Fix for #339

### CODE REVIEW

File: challenge_gpsr/src/gpsr.py [hunk 1]
```diff
@@ -186,12 +186,25 @@ def main():
...
+            
             nwc = NavigateToWaypoint(robot=robot,
-                                     waypoint_designator=EntityByIdDesignator(robot=robot,
-                                                                              id=knowledge.exit_waypoint),
-                                     radius = 0.3)
-            nwc.execute()
+                                             waypoint_designator=EntityByIdDesignator(robot=robot,
+                                                                                      id=knowledge.exit_waypoint),
+                                             radius = 0.3)
             robot.speech.speak("Thank you very much, and goodbye!", block=True)
```
Comment (LoyVanBeek) [inline challenge_gpsr/src/gpsr.py]: Should also be in the 'yes' condition

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek is the new version already correct?

**Comment by MatthijsBurgh** [comment]
@rokusottervanger seems to work, but AMIGO is not driving back before asking if challenge is over, because that is done at the start of the loop.

**Comment by MatthijsBurgh** [comment]
@rokusottervanger should we merge before Japan?

**Comment by MatthijsBurgh** [comment]
I suggest to not merge this PR. Do you agree @rokusottervanger?

**Comment by rokusottervanger** [comment]
I agree.

**Pull request closed without merging**